### PR TITLE
Update Configuration for Openvpn245

### PR DESCRIPTION
**Pull request by pmelange**
The comp-lzo option is no longer valid. Now the compress option must be used. Unfortunately having something analgous to comp-lzo=no is not supported.

The changes on the branch include the necessary compile option changes to include liblzo and updating the feed for the firmware-packages repo. There is also a PR in firmware-packages.

See issue freifunk-berlin/firmware#580.

### CODE REVIEW

File: feeds.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 src-git packages https://github.com/openwrt/packages.git^327a03e8f0c61f8ee98adbee5bd37ce7bccbb049
 src-git luci https://github.com/openwrt/luci.git^5ec72617eede467c932642b60291d627fca70342
 src-git routing https://github.com/openwrt-routing/packages.git^1b9d1c419f0ecefda51922a7845ab2183d6acd76
-src-git packages_berlin https://github.com/freifunk-berlin/firmware-packages.git^b0c64387a8bb5894b7efeaeff75b389009b656bb
+src-git packages_berlin https://github.com/freifunk-berlin/firmware-packages.git^317b65600e830be1d9f0e751cb5bbda137aa2540
 src-git gluon https://github.com/freifunk-gluon/packages.git^c34d129afb48f225741b683127beee6ba3c169d2
```
Comment (SvenRoederer) [inline feeds.conf]: normally the firmware master-branch only references to the master-branch of the packages-repo

Comment (pmelange) [reply feeds.conf]: Yes, and when the openvpn245 branch in the packages-repo is merged, then this merge should end up pointing to the master branch, right?

Shall I drop the commit to feeds.conf?

**Comment by pmelange** [comment]
A new commit has been added to openwrt-18.06
https://github.com/openwrt/openwrt/commit/d5afaa41148b363128626135bfb73b156887cddb
this may make this PR unneeded

**Comment by pmelange** [comment]
This is no longer needed as of https://github.com/openwrt/openwrt/commit/d5afaa41148b363128626135bfb73b156887cddb

**Pull request closed without merging**